templates/hp_procurve_show_interfaces_brief.textfsm

```diff
@@ -0,0 +1,31 @@
+Value PORT (\S+)
+Value TYPE (\S+|\s+)
+Value INTRUSION_ALERT (Yes|No)
+Value ENABLED (Yes|No)
+Value STATUS (Up|Down)
+Value MODE (\S+|\s+)
+Value MDI_MODE (\S+|s+)
+Value FLOW_CTRL (on|off)
+Value BCAST_LIMIT (\d+)
+
+Start
+  ^\s+Status.*Status\s*
+  ^\s*$$
+  ^\s.*Flow\s*
+  ^\s+Port.*Ctrl\s*
+  ^\s+-+\s+-+\s\+\s+\-+\s+-+\s+\-+\s+-+\s+-+\s+-+\s*$$ -> showintbrief1
```

Comment:
```suggestion
  ^\s+Port\s+Type\s+\|\s+Alert\s+Enabled\s+Status\s+Mode\s+Mode\s+Ctrl\s*$$ -> showintbrief1
```
This is to ensure we are capturing the fields we think we are (if we had the same output, but some fields had order changed, it might still record properly, but have incorrect key/value mappings)